pennylane_qiskit/converter.py

```diff
@@ -61,12 +61,11 @@ def _extract_variable_refs(params: Dict[Parameter, Any]) -> Dict[Parameter, qml.
     if params is not None:
         for k, v in params.items():
 
-            # Values can be arrays of size 1, need to extract the Python scalar
-            # (this can happen e.g. when indexing into a PennyLane numpy array)
-            if isinstance(v, np.ndarray):
-                v = v.item()
-
-            if isinstance(v, qml.variable.Variable):
+            if getattr(v, "requires_grad", True):
```

Comment:
It's a great appraoch to use `requires_grad`! One thing to note is that having `getattr(v, "requires_grad", True)` means that if `v` doesn't have a `requires_grad` attribute, we still perform the following operations. Is that what we'd want here? 🤔 

Also, it might be worth considering how we could support backwards compatibility so long as there's an option to use the old core.

Reply:
I think currently the policy is "Everything not explicitly marked with `requires_grad=False` is trainable"

Reply:
Yep, this is correct, but it's annoying so I'm hoping to switch it soon!